distributed/versions.py

```diff
@@ -2,35 +2,34 @@
 
 from __future__ import annotations
 
-import importlib
+import importlib.metadata
 import os
 import platform
 import struct
 import sys
-from collections.abc import Callable, Iterable
+from collections.abc import Iterable
 from itertools import chain
-from types import ModuleType
 from typing import Any
 
-required_packages = [
-    ("dask", lambda p: p.__version__),
-    ("distributed", lambda p: p.__version__),
-    ("msgpack", lambda p: ".".join([str(v) for v in p.version])),
-    ("cloudpickle", lambda p: p.__version__),
-    ("tornado", lambda p: p.version),
-    ("toolz", lambda p: p.__version__),
+required_packages: list[str] = [
```

Comment:
Does mypy ask for this explicit declaration? I'm surprised

Reply:
It doesn't. I just added it for documentation when the type was more complex (tuple involving callables, etc.) and didn't fully remove it. I still kinda like declarations like this for documentation purposes (though in this case I agree it doesn't seem necessary).